pnp-helper/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>iot-sdk-java</artifactId>
+        <groupId>com.microsoft.azure.sdk.iot</groupId>
+        <version>0.26.0</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>pnp-helper</artifactId>
+    <dependencies>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.8.1</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.microsoft.azure.sdk.iot</groupId>
+            <artifactId>iot-service-client-preview</artifactId>
```

Comment:
I'm not very sure of this in java. The IDE automatically added 1.0.1 but should we be using the latest? All the existing flattened.pom have this value and other places I see this:
`<dependency>
            <groupId>com.microsoft.azure.sdk.iot</groupId>
            <artifactId>iot-service-client-preview</artifactId>
            <version>${iot-service-client-version}</version>
        </dependency>`

Reply:
yes, 1.0.1 is the latest: https://mvnrepository.com/artifact/com.microsoft.azure.sdk.iot/iot-service-client-preview/1.0.1